client/src/components/screens/Patient/style.scss

```diff
@@ -209,12 +209,12 @@
         }
 
     }
-    .normal{
-        color: $green-7;
-    }
-    .abnormal{
+    .clinical__value--abnormal{
```

Comment:
It would be more compact if create create `.clinical` and add `&__value { &--abnormal{...}...} &__title{}` inside